packages/kbn-securitysolution-es-utils/src/delete_index_template/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { ElasticsearchClient } from '../elasticsearch_client';
+
+export const deleteIndexTemplate = async (
```

Comment:
Since these are public functions of the es-utils package, the linter is complaining that they lack JSDoc comments (which is subsequently lowering the "quality" rating of the package. It's not something you need to fix as part of this PR, but it's good to keep in mind moving forward and something we should address more holistically at a future time.